src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -260,3 +261,102 @@ by setting the parameter `elasticsearch.index.mailbox.name` to the name of your
 re-creates index upon restart
 
 _Note_: keep in mind that reindexing can be a very long operation depending on the volume of mails you have stored.
+
+## Setting Cassandra user permissions
+
+With a Cassandra server that serving multiple applications including James. 
+The keyspace of each James server need to be protected from another, 
+that also means, by the credential provided for a James server, 
+I cannot use it to access or modify other keyspaces. 
+
+We recommend you to not use the default Cassandra credential and provide 
+a different one with a minimized permissions for each application. 
+
+### Prerequisites
+
+We're gonna use the super user to create roles and grant permissions for them. 
+To do that, Cassandra requires you to login via username/password authentication 
+and enable granting in cassandra configuration file.
+
+For example:
+```
+echo -e "\nauthenticator: PasswordAuthenticator" >> /etc/cassandra/cassandra.yaml
+echo -e "\nauthorizer: org.apache.cassandra.auth.CassandraAuthorizer" >> /etc/cassandra/cassandra.yaml
+```
+
+### Prepare Cassandra roles & keyspaces for James  
+
+#### Create a role
+
+Visit [cassandra documentation](http://cassandra.apache.org/doc/latest/cql/security.html) section `CREATE ROLE` for more information
+
+E.g.
+```
+CREATE ROLE linagora WITH PASSWORD = 'linagora' AND LOGIN = true;
```

Comment:
Is there any way to not give the password in the CQL query but be prompt for it instead?

Reply:
There's no, prompting is only available in logins.